### PR TITLE
Leu1 card

### PR DESCRIPTION
**Pull request by leu1**
added my card to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <div class="container">
       <div class="row">
         <div class="column">
-          <!-- Night Mode Creation -->
+          <!-- Night Mode Creation --
```
Comment (malakada) [inline index.html:21]: Oops! Looks like you accidentally deleted the `>` character. 😅 

No worries, these things happen all the time. It's part of why code review exists! 🎉 

If this doesn't get replaced, it breaks the entire document for everyone, as the web browser will now think that everything until the next properly-closed comment is a comment. 😞 

Please make an extra commit to replace this.

File: index.html [hunk 2]
```diff
@@ -113,6 +113,31 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (malakada) [inline index.html:116]: Uh oh, looks like this was place in the wrong place.

It's not your fault, there was another card in the wrong spot near by too.

You actually want to put this in [just below here](https://github.com/Syknapse/Contribute-To-This-Project/pull/3668/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L147-L151). Can you please move it down? 

Here's a screenshot to make it easier to see where I'm talking about:

image

Sorry about that, but other than that the card itself looks great! 🎉

**Review by malakada — CHANGES_REQUESTED** [review]
Hey @leu1, thanks for taking the time to contribute! 🎉 

You're super close, I just have two changes that I noted inline in your code. Basically, you accidentally deleted a character and we need to move your card down to the right section.

😅 

No worries, and looking forward to seeing your card merged in! 💪

File: index.html (hunk 2 above)
Comment (leu1) [reply index.html:116]: corrected, thank you.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@leu1, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community

**Pull request merged**